### PR TITLE
[fix](test) fix test_create_table_without_distribution.groovy bug

### PR DESCRIPTION
**Pull request by feiniaofeiafei**
This pull request addresses issues in test_create_table_without_distribution.groovy by adding a mustContain function. The function checks if a string contains a specified substring, rather than verifying string equality, to prevent test failures caused by environmental variations.

### CODE REVIEW

**Comment by feiniaofeiafei** [comment]
run buildall

File: regression-test/suites/ddl_p0/test_create_table_without_distribution.groovy [hunk 1]
```diff
@@ -37,7 +38,6 @@ suite("test_create_table_without_distribution") {
     insert into test_create_table_without_distribution values(1,2);
     """
     qt_test_select_old_planner "select * from test_create_table_without_distribution;"
-    qt_test_show_old_planner " show create table test_create_table_without_distribution;"
-
-
+    def res2 = sql "show create table test_create_table_without_distribution;"
+    assertEquals(res2[0][1].contains("DISTRIBUTED BY RANDOM BUCKETS AUTO"), true)
```
Comment (morrySnow) [inline regression-test/suites/ddl_p0/test_create_table_without_distribution.groovy:42]:
```suggestion
    logger.info(res2[0][1])
    assertTrue(res2[0][1].contains("DISTRIBUTED BY RANDOM BUCKETS AUTO"))
```

Comment (924060929) [reply regression-test/suites/ddl_p0/test_create_table_without_distribution.groovy:42]: add  `assertContains` method to plugin, and print the return result to log.

for example
```groovy
assertContains(res2.toString(), "DISTRIBUTED BY RANDOM BUCKETS AUTO")
```

plugin example: https://github.com/apache/doris/blob/master/regression-test/plugins/plugin_check.groovy

**Comment by feiniaofeiafei** [comment]
run buildall

**Review by morrySnow — APPROVED** [review]

**Review by 924060929 — APPROVED** [review]

File: regression-test/plugins/plugin_must_contains.groovy [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    try {
+        assert str1.contains(str2)
+        logger.info("Assertion passed: '${str1}' contains '${str2}'")
+    } catch (AssertionError e) {
+        logger.error("Assertion failed: '${str1}' does not contain '${str2}'")
+        throw e
+    }
+    return true
+}
+logger.info("Added 'assertContains' function to Suite")
```
Comment (924060929) [inline regression-test/plugins/plugin_must_contains.groovy:30]: nit: change `assertContains` to `mustContain` in next pr

Comment (924060929) [inline regression-test/plugins/plugin_must_contains.groovy:30]: nit: change `assertContains` to `mustContain` in next pr

**Review by 924060929 — APPROVED** [review]

File: regression-test/plugins/plugin_must_contains.groovy (hunk 2 above)
Comment (feiniaofeiafei) [reply regression-test/plugins/plugin_must_contains.groovy:30]: ok~

**Pull request merged**